lightning/src/chain/keysinterface.rs

```diff
@@ -125,15 +151,16 @@ impl Readable for SpendableOutputDescriptor {
 			}),
 			1u8 => Ok(SpendableOutputDescriptor::DynamicOutputP2WSH {
```

Comment:
Somewhat unrelated to the diff itself, but as this keeps getting changed, how will we start ensuring backwards compatibility or migration capabilities in the future?

Reply:
Yes it would be great to have them more standardized, ideally as https://bitcoinops.org/en/topics/output-script-descriptors/

Reply:
Part of points to address in the future